core/definitions/src/protocols/governance/globalAccountant.ts

```diff
@@ -0,0 +1,39 @@
+import type { Chain, LayoutItem, MapLevel, Network } from "@wormhole-foundation/sdk-base";
+import type { EmptyPlatformMap } from "../../protocol.js";
+
+import { constMap } from "@wormhole-foundation/sdk-base";
+import "../../registry.js";
+declare module "../../registry.js" {
+  export namespace WormholeRegistry {
+    interface ProtocolToInterfaceMapping<N, C> {
+      GlobalAccountant: GlobalAccountant<N, C>;
+    }
+    interface ProtocolToPlatformMapping {
+      GlobalAccountant: EmptyPlatformMap<"GlobalAccountant">;
+    }
+  }
+}
+
+export type ModificationKind = "Add" | "Subtract" | "Unknown";
+export type ModificationKindEnum = 1 | 2 | 3;
+
+export const modificationKinds = [
+  [ "Add", 1 ],
+  [ "Subtract", 2 ],
+  [ "Unknown", 3 ],
+] as const satisfies MapLevel<ModificationKind, ModificationKindEnum>;
+
+export const modificationKindToEnum = constMap(modificationKinds);
+export const enumToModificationKind = constMap(modificationKinds, [1, 0]);
```

Comment:
I think this could be sort of reversed, declaring the `modificationKinds` with types of `MapLevel<string, number>` and then deriving the types of `ModificationKind` and `ModificationKindEnum` from the constmap produced

This has the advantage of defining the values in a single place so they never get out of sync